### PR TITLE
Add demo environment instructions and docker-config files

### PR DESCRIPTION
**Pull request by bbl232**
**What type of PR is this?**
/kind documentation

/area proposals

**What this PR does / why we need it**:
This PR adds a docker-compose.yml file that new users can use to quickly stand up and test out falco, falcosidekick, and falcosidekick-ui.

**Which issue(s) this PR fixes**:
None

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NO

```release-note
NONE
```

### CODE REVIEW

**Comment by leogr** [comment]
Nice!

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Considerations and guidance for Falco adopters:
 
 5. Integrate with output destinations: Integrate Falco with SIEM, data lake systems, or other preferred output destinations to establish a robust foundation for comprehensive data analysis and enable effective incident response workflows.
 
+### Demo Environment
+A demo environment is provided via a docker-compose file that can be started on a docker host which includes falco, falcosidekick, falcosidekick-ui and its required redis database. For more information see the [docker-compose section](docker/docker-compose/)
```
Comment (FedeDP) [inline README.md:47]:
```suggestion
### Demo Environment

A demo environment is provided via a docker-compose file that can be started on a docker host which includes falco, falcosidekick, falcosidekick-ui and its required redis database. For more information see the [docker-compose section](docker/docker-compose/)
```

Vertical spacing to follow other markdown sections formatting.

File: docker/docker-compose/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Warning
+This environment is provided for demonstration purposes only and does not represent a production ready deployment of falco
```
Comment (FedeDP) [inline docker/docker-compose/README.md:2]:
```suggestion
# Warning

This environment is provided for demonstration purposes only and does not represent a production ready deployment of falco
```
As above, vertical spacing :) Same for other sections.

File: docker/docker-compose/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Components
+The components that this docker-compose file spins up are falco, falcosidekick, falcosidekick-ui and a redis database
```
Comment (FedeDP) [inline docker/docker-compose/README.md:5]: I'd use some markdown links here, like: 
```suggestion
The components that this docker-compose file spins up are [Falco](https://falco.org/), [falcosidekick](https://github.com/falcosecurity/falcosidekick), [falcosidekick-ui](https://github.com/falcosecurity/falcosidekick-ui) and a [redis](https://redis.io/) database.
```

File: docker/docker-compose/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+# Running
+To start this environment run `docker-compose up`
```
Comment (FedeDP) [inline docker/docker-compose/README.md:8]:
```suggestion
To start this environment run `docker-compose up`.
```

File: docker/docker-compose/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+Note: You may need to use sudo for falco to start correctly.
```
Comment (FedeDP) [inline docker/docker-compose/README.md:9]:
```suggestion
Note: You may need to use sudo for Falco to start correctly.
```

File: docker/docker-compose/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+# Cleaning up
+To clean up run `docker-compose rm`
```
Comment (FedeDP) [inline docker/docker-compose/README.md:12]:
```suggestion
# Cleaning up

To clean up run `docker-compose rm`.
```

**Comment by FedeDP** [comment]
/milestone 0.39.0
Thanks for this PR, I really like the idea!

**Review by FedeDP — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**